README.md

```diff
@@ -76,6 +81,19 @@ using an array of gvcfs as input.*
   The dynamic scatter interval creating was optimized for genomes.  The scattered SNP 
   VariantRecalibration may fail because of two few "bad" variants to build the negative model. 
   Also, apologies that the logging for SNP recalibration is overly verbose.
+- No allele subsetting for the JointGenotyping workflow
```

Comment:
With respect to the caveat above "The current version of the posted...": the production version should have a switch based on the number of input GVCFs so I don't think this is an issue anymore.  Also we have tested on exomes, but it will require a different interval list.  Alas, the VariantRecalibration failure I put in is only for allele-specific filtering, so that might still be relevant.  Are allele-specific annotations on be default in our GVCF pipelines?

Reply:
Yes, looks like its enabled by default. [source](https://github.com/gatk-workflows/gatk4-germline-snps-indels/blob/30d95c15f0466c4b58341ce2d0abac1befddb2a7/JointGenotyping.wdl#L110) 
Does this bullet point need to be edited?

Reply:
Nope, that can stay.  I meant allele-specific annotations in the HaplotypeCaller workflow, which you did in new commit.